src/bin/pg_autoctl/system_utils.c

```diff
@@ -73,26 +74,33 @@ get_system_info_linux(SystemInfo *sysInfo)
 
 
 /*
- * FreeBSD, OpenBSD, and darwin use the sysctlbyname(3) API.
+ * FreeBSD, OpenBSD, and darwin use the sysctl(3) API.
  */
 #if defined(__APPLE__) || defined(BSD)
 static bool
 get_system_info_bsd(SystemInfo *sysInfo)
 {
 	unsigned int ncpu = 0;      /* the API requires an integer here */
+	int mib[2];
```

Comment:
What about using the following style, that would look more like what we use in the rest of the codebase?

``` c
    int ncpuMIB[2] = { CTL_HW, HW_NCPU };
```